src/generator.ts

```diff
@@ -101,7 +101,7 @@ export interface GeneratorOptions {
   /**
    * The provider to use for top-level (i.e. root package) installs if there's no context in the inputMap. This can be used to set the provider for a new import map. To use a specific provider for an install, rather than relying on context, register an override using the 'providers' option.
    *
-   * Supports: 'jspm.io' | 'jspm.io#system' | 'nodemodules' | 'skypack' | 'jsdelivr' | 'unpkg' | 'esm.sh';
+   * Supports: 'jspm.io' | 'denoland' | 'nodemodules' | 'skypack' | 'jsdelivr' | 'unpkg' | 'esm.sh';
```

Comment:
I changed it for `defaultProvider` not the registry 👍

Reply:
I don't think this works actually, since `denoland` is not the provider. The provider is `deno` and the registry is either `deno` or `denoland`. I think the mistake in the discussion was mixing up the registry and provider terms, I meant to say that it's the denoland registry not the deno registry that should be used.

I believe the `denoland` provider is an error, since it is not registered here - https://github.com/jspm/generator/blob/main/src/providers/index.ts#L55.